core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts

```diff
@@ -11,6 +11,22 @@ const API_ENDPOINT = '/api/v1/experiments';
 export class DotExperimentsService {
     constructor(private readonly http: HttpClient) {}
 
+    /**
+     * Add a new experiment
+     * @param  experiment
+     * @returns Observable<DotExperiment[]>
+     * @memberof DotExperimentsService
+     */
+    add(
+        experiment: Pick<DotExperiment, 'pageId' | 'name' | 'description'>
+    ): Observable<DotExperiment[]> {
+        const URL = `/api/v1/experiments`;
```

Comment:
I think we don't need this constant since it is used in one place.

Reply:
Deleted the constant of all the methods.